deployment_scripts/deploy_windows.ps1

```diff
@@ -8,6 +8,9 @@ param(
     [Parameter(Mandatory = $false, Position = 2)]
     [Bool]
     $agents = $true
+    [Parameter(Mandatory = $false, Position = 3)]
+    [System.String]
+    $version = "v1.7.0",
```

Comment:
Why 1.7? Why are we not querying for latest release?

Reply:
I made the changes in the config file for the latest, v1.7.0 isn't a parameter, I have just set it to define the argument type. And when we specifically provide the version argument, it will then only download that particular version. Otherwise it's going to download the latest.